### DISCUSSION TITLE
HOW TO DEBUG AN ERROR AFTER RUNNING A SIMULATION

### DISCUSSION THREAD

**Discussion by Joel040698** [OpenModelica/OpenModelica#12067] [Q&A]
I get this error after running a simulation of a hybrid system

*** Warning in Nuclear_Renewable_HES.dataCenterContinuousTimeControl.fan: Mover has per.etaMotMet=.Efficiency_MotorPartLoadRatio or per.etaMotMet=.GenericCurve.
This requires per.WMot_nominal or per.pressure to be provided. Because neither is provided,
the model overrides this setting and by using per.etaMotMet=.NotProvided.
Also consider using models under Movers.Preconfigured which autopopulate a pressure curve.

**Comment by adeas31** [comment]
https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/debugger.html#the-equation-based-debugger

**Comment by casella** [comment]
@Joel040698 this error message seems to me pretty library-specific, it's probably generated by some assert. I would suggest you to contact the library developers and ask them about it.
